path_static_creds_test.go

```diff
@@ -121,3 +121,18 @@ func TestCreds(t *testing.T) {
 		}
 	})
 }
+
+func readStaticCred(t *testing.T, b *backend, storage logical.Storage, roleName string) *logical.Response {
```

Comment:
added in the helper since it did not exist in this branch and it was being used in the test